vividus-plugin-rest-api/src/main/java/org/vividus/steps/api/HttpResponseValidationSteps.java

```diff
@@ -383,6 +388,39 @@ public void verifyArchiveContainsEntries(List<NamedEntry> parameters)
                         hasItem(expectedName)));
     }
 
+    /**
+     * Waits for a specified amount of time until HTTP response code is equal to what is expected.
+     * <p>
+     * <b>Actions performed:</b>
+     * </p>
+     * <ul>
+     * <li>Execute sub-steps</li>
+     * <li>Check if HTTP response code is equal to what is expected</li>
+     * </ul>
+     *
+     * @param responseCode for example 200, 404
+     * @param duration     Time duration to wait
+     * @param retryTimes   How many times request will be retried; duration/retryTimes=timeout between requests
+     * @param stepsToExecute Steps to execute at each wait iteration
+     */
+    @When("I wait for response code `$responseCode` for `$duration` duration retrying $retryTimes times"
```

Comment:
As I can see this step is not documented, so in case of step is changing and the documentation is missing, it should be added within the PR with changes